sigstore/_cli.py

```diff
@@ -536,10 +547,26 @@ def _verify(args: argparse.Namespace) -> None:
     elif args.rekor_url == DEFAULT_REKOR_URL:
         verifier = Verifier.production()
     else:
-        # TODO: We need CLI flags that allow the user to figure the Fulcio cert chain
-        # for verification.
-        args._parser.error(
-            "Custom Rekor and Fulcio configuration for verification isn't fully supported yet!",
+        if not args.certificate_chain:
+            args._parser.error(
+                "Custom Rekor URL used without specifying --certificate-chain"
+            )
+
+        try:
+            certificate_chain = _split_certificate_chain(
+                args.certificate_chain.read_text()
+            )
+        except SplitCertificateChainError as error:
+            args._parser.error(f"Failed to split certificate chain: {error}")
```

Comment:
Just to make it a little more generic:

```suggestion
            args._parser.error(f"Failed to parse certificate chain: {error}")
```